### PR TITLE
Ansible: add cluster configuration

### PR DESCRIPTION
**Pull request by TanguyRaufflet**
It is now possible to configure the cluster, install the cockpit plugins and test them from a fresh install of the Seapath Debain distribution.

### CODE REVIEW

File: ansible/deploy_cluster_test_plugins.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# SPDX-License-Identifier: Apache-2.0
+
+---
+# ansible_repository_path can be set in the inventory or via the argument "-e"
+- import_playbook: "{{ ansible_repository_path | default('../../../..') }}/playbooks/cluster_setup_debian.yaml"
+
+- name: Configure the cluster for Cockpit plugins testing
+  hosts:
+      - cluster_machines
+      - localhost
```
Comment (dupremathieu) [inline ansible/deploy_cluster_test_plugins.yml]: Do not run this on localhost.

File: ansible/deploy_cluster_test_plugins.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+      - name: Cockpit plugins installation (cluster machines)
+        become: true
+        block:
+          - include_tasks: roles/cockpit_plugins_cluster_setup/tasks/deploy_plugins_cluster.yml
+        when: inventory_hostname in groups['cluster_machines'] and cockpit_status.rc == 0
+
+        # For Selenium tests purposes
+      - name: Shutdown one of the host
+        block:
```
Comment (dupremathieu) [inline ansible/deploy_cluster_test_plugins.yml]: block is unnecessary here.

File: ansible/deploy_cluster_test_plugins.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          - command: "shutdown now"
```
Comment (dupremathieu) [inline ansible/deploy_cluster_test_plugins.yml]: How do you reboot the nuc3 when the test is ended?

File: ansible/roles/cockpit_plugins_cluster_setup/tasks/build_plugins.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2024, SFL (https://savoirfairelinux.com)
```
Comment (dupremathieu) [inline ansible/roles/cockpit_plugins_cluster_setup/tasks/build_plugins.yml:1]: Split the build part from the deployment part.
I preferred to use a script for the cockpit generation rather of using Ansible

**Review by dupremathieu — CHANGES_REQUESTED** [review]

File: ansible/deploy_cluster_test_plugins.yml (hunk 3 above)
Comment (TanguyRaufflet) [reply ansible/deploy_cluster_test_plugins.yml]: The host can be reboot using wakeonlan with the MAC address. This information has been added to the readme.

**Review by dupremathieu — APPROVED** [review]

**Pull request merged**